### DISCUSSION TITLE
Checkbox commands

### DISCUSSION THREAD

**Discussion by modem7** [renovatebot/renovate#17083] [Suggest an Idea]
Following on from https://github.com/renovatebot/renovate/discussions/15221#discussioncomment-3359340.

Some other tools have the ability to affect the PR via a command. 

E.g. 

`merge` will merge this PR after your CI passes on it
`squash and merge` will squash and merge this PR after your CI passes on it

Would it be possible to have a checkbox in the Dashboard/PR for the commands above so that it also deletes the branch after it's done, making things a bit more streamlined?

This would speed up operations when there are multiple PR's that require closing (e.g. when taking over a project/forks etc) where a CI/CD system may not (yet) be in place for whatever reason.

**Comment by rarkins** [comment]
Which platform are you using, e.g. github, gitlab, etc?

**Reply by modem7** [reply]
Just Github atm

**Reply by rarkins** [reply]
What's your reason for not using GitHub's own automerge feature? e.g. technical limitation or commercial (below)?

![image](https://user-images.githubusercontent.com/6311784/184114871-cd86c304-60f9-44f1-b66f-bed9d19b1219.png)

**Comment by modem7** [comment]
In this particular case, as some of the repos are fresh forks/projects or have been taken over, the CI/CD haven't been setup (especially for single person projects etc), so automerging is risky in case something breaks without a review.

**Reply by rarkins** [reply]
But you can click on GitHub automerge after review
